### PR TITLE
fix: skip lookups no currentVersion or lockedVersion

### PR DESCRIPTION
**Pull request by rarkins**
## Changes:

Shortcuts lookups if currentVersion is null

## Context:

Error in the app, in one private repo.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/process/lookup/index.ts [hunk 1]
```diff
@@ -254,11 +254,26 @@ export async function lookupUpdates(
...
+              currentDigest,
+              currentValue,
+              datasource,
+              depName,
+              lockedVersion,
+              packageFile,
+            },
+            'No dep currentVersion or lockedVersion'
+          );
+          continue;
```
Comment (viceice) [inline lib/workers/repository/process/lookup/index.ts:270]: Will this skip docker digest updates? There `currentVersion` can be undefined when implicit latest tag is used.

Comment (rarkins) [reply lib/workers/repository/process/lookup/index.ts:270]: The problem occurs when `getUpdateType(config, versioning, currentVersion, newVersion)` is called and `currentVersion` is null. I'm not 100% sure of the best place to do this fix.

Comment (viceice) [reply lib/workers/repository/process/lookup/index.ts:270]: So we should at least do a dry run against a repo with docker dicgest updates to validate

Comment (rarkins) [reply lib/workers/repository/process/lookup/index.ts]: Tried another approach

**Pull request merged**